app/policies/district_policy.rb

```diff
@@ -0,0 +1,52 @@
+class DistrictPolicy < ApplicationPolicy
+  def index?
+    user.owner? || user.organization_owner?
+  end
+
+  def show?
+    user.owner? || [:organization_owner, :supervisor, :analyst].map { |role| admin_can_access?(role) }.any?
+  end
+
+  def create?
```

Comment:
Unnecessary

Reply:
Removed all except for `graphics` (which might be used later when Whatsapp graphics is re-enabled, if I read it correctly).